py/dml/ctree.py

```diff
@@ -379,6 +391,12 @@ def control_flow(self):
 
 mkThrow = Throw
 
+class UnrolledBreak(Goto):
+    def control_flow(self):
+        return ControlFlow(fallthrough=True)
```

Comment:
should be `br=True`? Test needed (see T_ENORET)

Reply:
I had tested with `br=True`, but that resulted in broken generated C. Same thing with `fallthrough=False`.
I agree, test is warranted.

Reply:
Indeed, this incorrectl y reports `ENORET`:
```
method m10() -> (int) {
    try {
        #foreach b in ([dev]) {
            break;
            // this statement is dead...
            throw;
        }
        return 0;
    } catch {
        // .. so this block is also dead ..
    }
    // .. so finally, this point is unreachable and we should get no error
}
```
The obvious way to fix this requires the addition of a DML statement class corresponding to `#foreach` statements.

Reply:
Aforementioned fix implemented.